Formula/cgal.rb

```diff
@@ -1,17 +1,10 @@
 class Cgal < Formula
   desc "Computational Geometry Algorithms Library"
   homepage "https://www.cgal.org/"
-  url "https://github.com/CGAL/cgal/releases/download/v5.3-beta1/CGAL-5.3-beta1.tar.xz"
-  sha256 "0b9df5df2ee19d6298c9cac95fbbdd3ea56a665c29a48b0fe6a7bfa47fc3ea7f"
+  url "https://github.com/CGAL/cgal/releases/download/v5.3/CGAL-5.3.tar.xz"
+  sha256 "2c242e3f27655bc80b34e2fa5e32187a46003d2d9cd7dbec8fbcbc342cea2fb6"
   license "GPL-3.0-or-later"
 
-  bottle do
-    sha256 cellar: :any_skip_relocation, arm64_big_sur: "39d8e1234f08860e0fd53131654a3986e093e21a2ce0a268241acdf119d82e12"
-    sha256 cellar: :any_skip_relocation, big_sur:       "25fdaa548ad0599f97ead357fa87b98e7074c90a4643e7e36f8cd84e15c0d5ed"
-    sha256 cellar: :any_skip_relocation, catalina:      "25fdaa548ad0599f97ead357fa87b98e7074c90a4643e7e36f8cd84e15c0d5ed"
-    sha256 cellar: :any_skip_relocation, mojave:        "25fdaa548ad0599f97ead357fa87b98e7074c90a4643e7e36f8cd84e15c0d5ed"
-  end
```

Comment:
Please don't touch the `bottle` block.

Reply:
It wouldn't install with it. I thought it was the CI that was generating it

Reply:
Also, please rebase with `git rebase` to get rid of the merge commit -- @BrewTestBot is unable to merge PRs with merge commits.

Reply:
You need to pass the `--build-from-source` flag to install it from source when the bottle block doesn't match the formula. Just please put the `bottle` block back.